exporter/fileexporter/config.go

```diff
@@ -29,6 +29,13 @@ type Config struct {
 
 	// Rotation defines an option about rotation of telemetry files
 	Rotation Rotation `mapstructure:"rotation"`
+
+	// MarshalType defines the type of marshaler which encodes telemetry data
+	// Options:
+	// - json[default]: Marshals to OTLP json bytes.
+	// - proto: Marshals to OTLP binary protobuf bytes.
+	// - protobuf: Marshals to OTLP binary protobuf bytes.
+	MarshalType string `mapstructure:"marshaler"`
```

Comment:
Users may be less familiar than us with the term "marshaler". What do you think about just calling this "format"?

Reply:
`format` is more appropriate